admission_control/admission_control_service/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 
 [dependencies]
 futures = "0.1.28"
-futures03 = { version = "=0.3.0-alpha.17", package = "futures-preview" }
+futures03 = { git = "https://github.com/rust-lang-nursery/futures-rs.git", rev = "cde791c00b8b9c4fd14a594855038a1bc4b6323e", package = "futures-preview" }
```

Comment:
I think we should file a bug to undo the git version once futures-preview publishes so this doesn't get forgotten. Do we when about when that would be?

Reply:
I already filed https://github.com/rust-lang-nursery/futures-rs/issues/1884 and it looks like we shouldn't have to wait too long